### PR TITLE
Add query support to structured

### PR DESCRIPTION
**Pull request by trescube**
Adds `query` parameter support to `/v1/search/structured`.  Most everything was in place before, this just adds `query` to analysis synthesis and adjusts tests to query module changes.  

Connected to pelias/pelias#483

### CODE REVIEW

File: sanitizer/_synthesize_analysis.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const _ = require('lodash');
 const text_analyzer = require('pelias-text-analyzer');
 
 const fields = {
+  'query': 'query',
```
Comment (dianashk) [inline sanitizer/_synthesize_analysis.js]: Can we call the param something other than query? Maybe poi_name or venue? Query is confusing I'm my opinion.

**Comment by dianashk** [comment]
After discussing in our team meeting, we think changing this to `name` makes sense here. We will continue to also search the categories for results, but the user doesn't need to be directly aware of that. When we officially release the `categories` parameter for search it will be supported in this endpoint as well, so it doesn't seem necessary to include a dedicated `category` parameter for the query as well.

**Comment by trescube** [comment]
Oh, sorry I missed this discussion since I was ooto.

**Comment by trescube** [comment]
After a team meeting on 1/11/2017, it was decided that the parameter name would be `venue`.

**Pull request merged**